postman-collection.json

```diff
@@ -0,0 +1,1119 @@
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Filter (url_contains_at_least_two)",
+					"protocolProfileBehavior": {
+						"disableBodyPruning": true
+					},
+					"request": {
+						"auth": {
+							"type": "oauth2",
+							"oauth2": [
+								{
+									"key": "clientSecret",
+									"value": "{{GOOGLE_CLIENT_SECRET}}",
+									"type": "string"
+								},
+								{
+									"key": "clientId",
+									"value": "{{GOOGLE_CLIENT_ID}}",
+									"type": "string"
+								},
+								{
+									"key": "authRequestParams",
+									"value": [
+										{
+											"key": "access_type",
+											"value": "offline",
+											"enabled": true,
+											"send_as": "request_url"
+										},
+										{
+											"key": "scope",
+											"value": "profile email",
+											"enabled": true,
+											"send_as": "request_url"
+										}
+									],
+									"type": "any"
+								},
+								{
+									"key": "accessTokenUrl",
+									"value": "{{base_url}}/api/v1/users/auth/google_oauth2/callback",
+									"type": "string"
+								},
+								{
+									"key": "redirect_uri",
+									"value": "{{base_url}}/api/v1/users/auth/google_oauth2/callback",
+									"type": "string"
+								},
+								{
+									"key": "tokenRequestParams",
+									"value": [],
+									"type": "any"
+								},
+								{
+									"key": "addTokenTo",
+									"value": "header",
+									"type": "string"
+								},
+								{
+									"key": "client_authentication",
+									"value": "body",
+									"type": "string"
+								},
+								{
+									"key": "grant_type",
+									"value": "authorization_code",
+									"type": "string"
+								},
+								{
+									"key": "authUrl",
+									"value": "https://accounts.google.com/o/oauth2/v2/auth",
+									"type": "string"
+								},
+								{
+									"key": "useBrowser",
+									"value": false,
+									"type": "boolean"
+								},
+								{
+									"key": "headerPrefix",
+									"value": "Bearer",
+									"type": "string"
+								},
+								{
+									"key": "state",
+									"value": "",
+									"type": "string"
+								},
+								{
+									"key": "scope",
+									"value": "",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "formdata",
+							"formdata": [
+								{
+									"key": "csv_file",
+									"type": "file",
+									"src": "/Users/tuubz/Desktop/keywords.csv"
+								},
+								{
+									"key": "client_id",
+									"value": "{{CLIENT_ID}}",
+									"type": "text",
+									"disabled": true
+								},
+								{
+									"key": "client_secret",
+									"value": "{{CLIENT_SECRET}}",
+									"type": "text",
+									"disabled": true
+								}
+							]
+						},
+						"url": {
+							"raw": "{{base_url}}/api/v1/search_results?filter[url_contains_at_least_two]=:",
```

Comment:
Following Olivier's suggestion https://github.com/nimblehq/ic-rails-huey-toby/pull/57#discussion_r1268879840, I updated the endpoint for more flexible, please help to update as well 😅